include/VecCore/Backend/Implementation.h

```diff
@@ -284,6 +285,21 @@ VECCORE_FORCE_INLINE VECCORE_ATT_HOST_DEVICE constexpr Bool_s EarlyReturnAllowed
 #endif
 }
 
+// An extension of EarlyReturnAllowed(), and new name is a reminder that it can be used for other purposes as well
+template <typename T, size_t simdSize = 1>
+VECCORE_FORCE_INLINE
+VECCORE_ATT_HOST_DEVICE
+constexpr Bool_s SIMDsizeUpTo()
```

Comment:
I agree with the naming issue.

Waht about ```ERVecSizeAtMost``` (instead of the longer EarlyReturnVectorSizeAtMost or EarlyReturnAllowedVectorSizeAtMost)